crates/pyo3/src/py_node.rs

```diff
@@ -168,6 +168,17 @@ impl SgNode {
     })
   }
 
+  fn field_children(&self, name: &str) -> Vec<SgNode> {
```

Comment:
Thanks! Another place we need to add is the pyi file. https://github.com/ast-grep/ast-grep/blob/efba7e5f30540f816bfa759ae4192b444d90f8d0/crates/pyo3/ast_grep_py/ast_grep_py.pyi#L56